### DISCUSSION TITLE
Sponsorship still in Pending state

### DISCUSSION THREAD

**Discussion by fmcarvalho** [community/community#82410] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have completed the required steps successfully to submit application to GitHub Sponsors. Yet my application is still in Pending state for weeks without any answer from Github. I did not received any email either from Github about this process.

![image](https://github.com/community/community/assets/578217/453755c4-bcd6-4732-a7cb-b173b23f7c1e)

**Comment by DHayes10** [comment]
Hi @fmcarvalho, 

We have forwarded your profile to the Sponsors team for approval. You should see an update within the next hour. If you have any other questions, feel free to reach back out. 

Thank you, 
GitHub
